examples/viewer.py

```diff
@@ -46,24 +46,43 @@ class HabitatSimInteractiveViewer(Application):
     DISPLAY_FONT_SIZE = 16.0
 
     def __init__(self, sim_settings: Dict[str, Any]) -> None:
+        self.sim_settings: Dict[str:Any] = sim_settings
+
+        self.enable_batch_renderer: bool = self.sim_settings["enable_batch_renderer"]
+        self.num_env: int = (
+            self.sim_settings["num_environments"] if self.enable_batch_renderer else 1
+        )
+
+        # Compute environment camera resolution based on the number of environments to render in the window.
+        surface_size: tuple(int, int) = (
+            self.sim_settings["window_width"],
+            self.sim_settings["window_height"],
+        )
```

Comment:
In the `replayer` I have the camera resolution as the configuration parameter, with the window size being inferred from that. Not sure which of the two is the better approach, but we should probably use the same in both :)

Reply:
For user-driven visualization purposes, I think it makes more sense to define a window size, then subdivide the surface based on the environment count. It saves a napkin calculation.

Eventually, this viewer will become an interface exposed to the lab. It will be up to the implementer to figure out what makes sense to them.